Training/1.0 Introduction.md

```diff
@@ -12,37 +12,69 @@ First we we learn a little bit about coding in Python, although many of you will
 List all the Data Types of Python with examples.  
 
 ```
-
+str:'hello'
+int:1
+float:2.3
+bool:True
+complex:1j
+list:[1,2,3]
+tuple:('a','s,'b')
+range:range(5)
+dict:{'name':'Murat','age':23}
+set:{"apple", "banana", "cherry"}
+frozenset:frozenset({"apple", "banana", "cherry"})
+bytes:b"Hello"
+bytearray:bytearray(5)
+memoryview:memoryview(bytes(5))
+NoneType:None
 ```
 
 ### Task 2: 
 
 Other useful ways of storing data are in List, Arrays and Tuples. Create one of each and assign them to the variables a, b and c respectively. 
 
 ```
-
+import numpy as np
+a=[1,2,3]
+b=np.array([[1,2,3],[3,4,5]])
+thistuple = ("apple", "banana", "cherry")
 ```
 
 ### Task 3: 
 
 We are also interested in becoming good software developers so we will need to use conditional, loops. Write an statement where the summation of a variable x+1 will be calculated if the value of x is greater than 2. 
 
 ```
-
+x=3
+if x>2:
+    summ=x+1
 ```
 
 Now incorporate a for loop to calculate the value of n(x+1) for n repetitions where n=3. Store each value of the for loop in an array named y.  
 
 ```
-
+import numpy as np
+y=np.array([])
+n=3
+x=2
+for i in range(n):
+    prd=i*(x+1)
+    y=np.append(y,prd)
 ```
 
 ### Task 4: 
 
 Finally we will have a look at creating functions. Functions allow us to compute processes much faster without having to repeat lines of code. Using your code from Task 3, create a function called 'my_cool_function', which takes in the value x and will compute the n(x+1) if x is greater than 2 and for n repetitions where values will be stored in an array named y. Your function will return y. 
 
 ```
-
+def my_cool_function(x,n):
+    if x>2:
+      import numpy as np
+      y=np.array([])
+      for i in range(n):
+          prd=i*(x+1)
+          y=np.append(y,prd)
+    return y
```

Comment:
Correct! Normally, it's good practice to import your libraries and modules at the top of your script.